src/receivers/ExpressReceiver.ts

```diff
@@ -33,6 +34,7 @@ export interface ExpressReceiverOptions {
   installationStore?: InstallProviderOptions['installationStore']; // default MemoryInstallationStore
   scopes?: InstallURLOptions['scopes'];
   installerOptions?: InstallerOptions;
+  router?: core.Router;
```

Comment:
`IRouter` should be a more relevant type for this argument as `express.Application` instance inherits the type not `core.Router`.

Reply:
ahh gotcha, thanks! Tbh I didn't know `IRouter` even existed 😅